src/ActorManager.cpp

```
@@ -557,25 +557,31 @@ namespace hdt
 				maxActiveSkeletons,
 				m_skeletons.size(),
 				averageProcessingTimeInMainLoop,
-				target_time);
+				maxBudgetTime);
 
 			if (m_autoAdjustMaxSkeletons) {
-				maxActiveSkeletons += target_time > averageProcessingTimeInMainLoop ? 2 : -2;
+				// Deadzones to prevent constantly switching back and fourth
+				if (averageProcessingTimeInMainLoop > maxBudgetTime) {
+					maxActiveSkeletons -= 2;
+				} else if (averageProcessingTimeInMainLoop < maxBudgetTime * 0.9f) {  // under 90% of budget
```

Clang (14.0.6)

[warning] floating point literal has suffix 'f', which is not uppercase

(readability-uppercase-literal-suffix)